SQL/Cleanup_patches/README.md

```diff
@@ -0,0 +1,3 @@
+This directory contains SQL patches written as a post-release clean-up. These patches can contain `ALTER TABLE`, `DELETE` and/or `DROP` statements.
+
+These patches should be run at the discretion of the projects with the full knowledge of the possible side effects of these.
```

Comment:
"of the patches" or "of these patches"

Reply:
I think this is also part of the first paragraph and not a second paragraph too

Reply:
I like "of these patches"

Reply:
Another option is to just leave off the "of these"